Atomic/atomic.py

```diff
@@ -254,6 +270,80 @@ def push(self):
 
                 prevstatus = status
 
+
+    def _pull_dockertar_layers(self, repo, imagebranch, temp_dir, input_layers):
+        layers = {}
+        next_layer = {}
+        top_layer = None
+        for i in input_layers:
+            layer = i.replace("/layer.tar", "")
+            layers[layer] = os.path.join(temp_dir, i)
+            with open(os.path.join(temp_dir, layer, "json"), 'r') as f:
+                json_layer = json.loads(f.read())
+                parent = json_layer.get("parent")
+                if not parent:
+                    top_layer = layer
+                next_layer[parent] = layer
+
+        layers_map = {}
+        enc = sys.getdefaultencoding()
+        for k, v in layers.items():
+            out = subprocess.check_output([ATOMIC_LIBEXEC + '/dockertar-sha256-helper',
+           v], stderr=DEVNULL)
```

Comment:
Should this be overridden via ENV variable or search for it locally as well as in /usr/libexec/atomic, so that we can run tests before installing the package.